addons/website/static/src/xml/website_form_editor.xml

```diff
@@ -39,6 +39,15 @@
                     <t t-call="website.form_field_description"/>
                 </div>
             </div>
+            <t t-elif="field.formatInfo.labelPosition == 'after'">
+                <div class="d-flex">
+                    <t t-out="0"/>
+                    <label class="s_website_form_label s_website_form_label_after" t-attf-style="width: #{field.formatInfo.labelWidth or '200px'}" t-att-for="field.id">
```

Comment:
` s_website_form_label_after ` :  unless I’m missing something, it looks like we’re creating a new class just to set its corresponding UI button to "active"...

Read my previous comment, you may be able to use `form-check-label` instead 😉